Frameworks/CoreText/DWriteWrapper.mm

```diff
@@ -190,7 +190,7 @@ static CFStringRef _CFStringFromLocalizedString(IDWriteLocalizedStrings* localiz
     NSDictionary* attribs = [ts->_attributedString attributesAtIndex:range.location effectiveRange:NULL];
 
     CGFloat fontSize = kCTFontSystemFontSize;
-    std::vector<wchar_t> familyName(c_defaultFontName, c_defaultFontName + wcslen(c_defaultFontName));
+    std::vector<wchar_t> familyName;
```

Comment:
derp, for some reason I completely missed that the below block was conditional

---
In reply to: [80733607](https://github.com/Microsoft/WinObjC/pull/1039#discussion_r80733607) [](ancestors = 80733607)